airflow/models/taskinstance.py

```diff
@@ -1762,19 +1767,18 @@ def filter_for_tis(
         tis: Iterable[Union["TaskInstance", TaskInstanceKey]]
     ) -> Optional[BooleanClauseList]:
         """Returns SQLAlchemy filter to query selected task instances"""
-        TI = TaskInstance
         if not tis:
             return None
         if all(isinstance(t, TaskInstanceKey) for t in tis):
-            filter_for_tis = ([and_(TI.dag_id == tik.dag_id,
-                                    TI.task_id == tik.task_id,
-                                    TI.execution_date == tik.execution_date)
+            filter_for_tis = ([and_(TaskInstance.dag_id == tik.dag_id,
+                                    TaskInstance.task_id == tik.task_id,
+                                    TaskInstance.execution_date == tik.execution_date)
                                for tik in tis])
             return or_(*filter_for_tis)
         if all(isinstance(t, TaskInstance) for t in tis):
-            filter_for_tis = ([and_(TI.dag_id == ti.dag_id,
-                                    TI.task_id == ti.task_id,
-                                    TI.execution_date == ti.execution_date)
+            filter_for_tis = ([and_(TaskInstance.dag_id == ti.dag_id,
+                                    TaskInstance.task_id == ti.task_id,
+                                    TaskInstance.execution_date == ti.execution_date)
```

Comment:
If we decide on long names then we should remove TR from global scope, shouldn't we?

Reply:
> Is this another attempt to fix cyclic imports? #9674

It was not supposed to be, but it turned out to be exactly this :).

Reply:
Yes.

Reply:
Ahh. I fixed it in a simple way I think :)